cypress/integration/find.js

```diff
@@ -40,5 +40,23 @@ describe('Find', function() {
 		assertQuery('&gh internet4000/radio4000', 'https://github.com/internet4000/radio4000')
 		assertQuery('&gh internet4000/radio4000', 'https://github.com/internet4000/radio4000')
 	})
+
+	it('has a method to replace a placeholder', () => {
```

Comment:
not sure how this test helps?

Reply:
i added it as placeholder, needs real tests but wan't sure how the method works

Reply:
alright!